Tools/wasm/wasi.py

```diff
@@ -16,7 +16,7 @@
 import tempfile
 
 
-CHECKOUT = pathlib.Path(__file__).parent.parent.parent
+CHECKOUT = pathlib.Path(__file__).parent.parent.parent.resolve()
```

Comment:
Without this change, we get paths with `..` like `cpython/Tools/wasm/../../cross-build/wasm32-wasi/` showing up. I'm surprised that pathlib returns `/a/b/..` when you ask for the parent of `/a/b`, that doesn't really seem like the best answer to me.

Reply:
> I'm surprised that pathlib returns `/a/b/..` when you ask for the parent of `/a/b`, that doesn't really seem like the best answer to me.

It's because it's a purely syntactic operation on the path to get the parent. If you want to pay the perf penalty of resolving `..` you need to call `resolve()`.